Dockerfile_site

```diff
@@ -42,13 +52,4 @@ RUN pip3 install --break-system-packages --upgrade "/code[gcp]" \
 
 EXPOSE 8082
 
-# TODO: config
-# TODO: service discovery
-RUN apt-get satisfy --yes --no-install-recommends \
```

Comment:
pretty sure these are actually required since the web server depends on e.g. chart.js and jquery